src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+        "Unavailable fields unchecked give users access to view system fields that are not available.",
+        "Also this was changed to give users easy access to enable/disable system fields based on the button options provided."
+      ],
+      "how": [
+        "By updating system field mapping logic to prevent Record ID from being disabled, this also applies to backend.",
+        "System view configurator with the unavailable badge was also updated.",
+        "Updated module dal settings to display button groups, with the logic to enable/disable fields that are related to a button"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/523/commits/07d41dcd8c1f40abf17f90868953eeb3603e0809" 
+      ]
+    },
+    {
+      "what": [
+        "the name of a page when was copied multiply times was changed to show the copy and after that the name of the page and a counter in brackets."
+      ],
+      "why": [
+        "to improve user experience.",
+        "If we copy a page 3-4 or more times, the naming gets confusing and too long."
+      ],
+      "how": [
+        "by filtering how many times a certain page with same name already exists and including 'copy of' prefix and number suffix.",
+        "Depending on the number of pages that match the same name, a counter is adjusted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/644/commits/cbaca62b474fac5e9a19e9c500104fb93bf0ac9d" 
+      ]
+    },
+    {
+      "what": [
+        "workflow iterators to treat limit 0 parameter as no limit set."
+      ],
+      "why": [
+        "to make workflow iterators consistent with other searching/filtering interactions."
+      ],
+      "how": [
+        "by changing the limit in the code."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/cc24e622a" 
+      ]
+    },
+    {
+      "what": [
+        "the record list filter component updated with a unique id, which was previously the block id, also updated with a style of position: initial to prevent the record filter pop over z index issues."     
+      ],
+      "why": [
+        "to fix the issue of filtering not working on modal or full screen magnification."
+      ],
+      "how": [
+        "by filter component using a unique id for the target value of the filter pop over."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/709/commits/0cee8ec88217959f40887df78a51f58da355ffa2" 
+      ]
+    },
+    {
+      "what": [
+        "the location of the attachment modal."     
+      ],
+      "why": [
+        "to enable previewing of attachments in private pages."
+      ],
+      "how": [
+        "by moving the attachment modal from Public/Pages/View to Namespace/View."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/6ade04484397285d541a2da1ff88e497c98717fb" 
+      ]
+    },
```

Comment:
I'd omit this; just some technical stuff.
You could put this under the generic UI changes/tweaks entry if you wish